flink-connector-akka/README.md

```diff
@@ -0,0 +1,45 @@
+# Flink Akka connector
+
+This connector provides a sink to [Akka](http://akka.io/) source actors in an ActorSystem.
+To use this connector, add the following dependency to your project:
+
+
+    <dependency>
+      <groupId>org.apache.bahir</groupId>
+      <artifactId>flink-connector-akka_2.11</artifactId>
+      <version>1.0.0-SNAPSHOT</version>
+    </dependency>
+    
+*Version Compatibility*: This module is compatible with Akka 2.0+.
+    
+## Configuration
+    
+The configurations for the Receiver Actor System in Flink Akka connector can be created using the `Configuration (org.apache.flink.configuration.Configuration)` object in Flink.
+    
+To enable acknowledgements, the custom configuration `akka.remote.auto-ack` can be used.
+
+The user can set any of the default configuration allowed by Akka as well as custom configuration allowed by the connector.
+   
+A sample configuration can be defined as follows:
+    
+    Configuration configuration = new Configuration();
+    configuration.setString("akka.loglevel", "INFO");
+    configuration.setString("akka.actor.provider", "akka.remote.RemoteActorRefProvider");
+    configuration.setString("akka.remote.netty.tcp.hostname", "127.0.0.1");
+    configuration.setString("akka.remote.enabled-transports", "[akka.remote.netty.tcp]");
+    configuration.setString("akka.remote.netty.tcp.port", "5150");
+    configuration.setString("akka.remote.log-sent-messages", "on");
+    configuration.setString("akka.remote.log-received-messages", "on");
+    configuration.setString("akka.remote.auto-ack", "on");    
```

Comment:
How can a user pass the `configuration` to the Akka source? Afaik its not possible because the open(Configuration c) is not really supported in the DataStream API of Flink.